user/serializers.py

```diff
@@ -41,9 +41,7 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _("User account is disabled.")
-                    raise serializers.ValidationError(
-                        msg, code="authorization"
-                    )
+                    raise serializers.ValidationError(msg, code="authorization")
```

Comment:
Ensure that the `authenticate` function is correctly configured to use the custom user model, as it defaults to using the username field. If your custom user model uses email for authentication, make sure the authentication backend is set up to handle this.